pkg/api/api.proto

```diff
@@ -203,6 +216,8 @@ message Environment {
   Config config = 2;
   map<string, Lock> locks = 3;
   map<string, Application> applications = 4;
+  uint32 distanceToUpstream = 5;
```

Comment:
If we are putting the `distanceToUpstream` in the `EnvironmentGroup`, do we need to duplicate it here ?